### ISSUE TITLE
charts improvements (esp. save chart button)

### ISSUE THREAD

**Issue by antonkulaga** [polynote/polynote#1034]
When I draw charts with seaborn and other libraries I do not have the button to download the chart, especially if the chart is svg-based. I suggest to:
1) consider download button
2) have an option to toggle on/off inline ploting (like %matplotlib inline )

**Comment by jonathanindig** [comment]
Hi @antonkulaga, thanks for the suggestion. 

1. Can you please provide a minimal example of the charts you're using and what sort of download button you'd like? Does seaborn provide a download button on other platforms (e.g., Jupyter)? If so, can you share an example of what that looks like? 

2. I'm assuming the current behavior of plotting is what you'd call "inline", right? What would you expect the behavior of non-inline plotting to look like? 

Thanks!

**Comment by antonkulaga** [comment]
In my case the most pain was with upset plot, that is matplitlib based ( https://github.com/jnothman/UpSetPlot ). You can reproduce any of their examples and see that you can download their output in jupyter but cannot in polynote (without going to debug mode to copy-paste svg content). Maybe polynote should detect svg in the output and generate a button to download it?

**Comment by jonathanindig** [comment]
@antonkulaga I don't see any download button in Jupyter. Is that coming from some sort of browser / Jupyter extension? It would be helpful if you could send a screenshot.

I do see that in Polynote the output is an SVG, but in Jupyter it is a PNG file. Since browsers allow you to right-click to save PNG files (but for some reason don't really let you do this with SVG files), I can see how that's a bit of a pain. Luckily, it's a very simple change to tell Polynote that matplotlib charts should be PNG files by default - we can also make this configurable. I created https://github.com/polynote/polynote/issues/1037 to track this request.

**Comment by antonkulaga** [comment]
@jonathanindig indeed, I did not notice that. Overall SVG is superior to png, but there should be  a button to save it. Is it possible to catch all SVG-s in the output and generate a button for it?

**Comment by jeremyrsmith** [comment]
@antonkulaga Doesn't your browser have a context menu "Save image"?

**Comment by antonkulaga** [comment]
not for svg

**Comment by jeremyrsmith** [comment]
Hmm, I wonder if this is due to SVG being directly embedded into the document rather than in an `<img>` tag? Maybe that would solve it?

If so, I don't think Polynote adding buttons everywhere is a particularly good solution, because lots of results have different meanings and different ways to download them (see for example vega specs, which add their own download buttons)

**Comment by antonkulaga** [comment]
yes, it is directly EMBED by matplotlib instead of the svg tag
